modules/bamtools/stats/main.nf

```diff
@@ -0,0 +1,35 @@
+process BAMTOOLS_STATS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::bamtools=2.5.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bamtools:2.5.1--h9a82719_9' :
+        'quay.io/biocontainers/bamtools:2.5.1--h9a82719_9' }"
+
+    input:
+    tuple val(meta), path(bam)
+
+    output:
+    tuple val(meta), path("*.stats"), emit: stats
+    path "versions.yml"             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    bamtools \\
+        stats \\
+        -in $bam \\
+        >${prefix}.bam.stats
+
```

Comment:
Is there any optional arguments for the tool at all?

Reply:
there isn't. there's an alternative input `-list` instead of `-in` if you have multiple bam files listed in a file, but I thought in an nf-core pipeline we would want to distribute the tasks.